### DISCUSSION TITLE
Tutorial on integrating custom blocks with Write mode

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#330] [Topic Ideas]
Considering work is expanding to bring a [Write mode to fruition](https://github.com/WordPress/gutenberg/issues/60021) that greatly simplifies the editing experience, we need developers to be aware of the impact on their respective blocks, whether one off custom blocks for clients or broader blocks tied into bigger plugins. As a result, I think a dedicated tutorial on the topic that also explains the goal of write mode would be very helpful to see early in the 6.8 release cycle.

**Comment by ndiego** [comment]
@annezazu, a dedicated article on content-only mode is already under second review here, #303. There could definitely be a follow-up early next year on whether Write mode is in a better place, but I wanted to get this info out as soon as possible so developers can opt into `contentOnly` as soon as 6.7 releases.

**Comment by annezazu** [comment]
Oh amazing! Thank you. I tried to find something similar but didn’t manage to spot that one. We can close this out or keep it open as Write evolves. I trust you all there.

**Comment by ndiego** [comment]
Once we are further along in the release cycle, I think it would still be great to have a dedicated tutorial on block development as it relates to Write mode. It will be a good follow-up to #303, and I'll plan on taking this on.

**Comment by bph** [comment]
Topic was approved and conversation continues on issue
- #335
